### ISSUE TITLE
Can we help maintain this repo?

### ISSUE THREAD

**Issue by alycedaw** [daltoniam/Starscream#962]
There are currently 30 PR's open and 127 issues, most of which don't have reply's. This is a fairly popular dependency, but there haven't been any updates for a few years. I would be happy to help maintain this repo although it would take me a bit to understand the project as a whole and come up to speed on how the underlying code works. 

Otherwise, has this project been abandoned since Apple release WebSocket support in Foundation using `URLSessionWebSocketTask`?

**Comment by sammycooks999** [comment]
@daltoniam Many would like this... Starscream lives on.. even in https://github.com/signalapp/Signal-iOS

**Comment by raymondwelch33** [comment]
At this point, I say we create a new repository, call it `Megatron` and begin messaging each contributor here and recruit them to help us further improve and extend `Starscream`.

**Comment by daltoniam** [comment]
Howdy y'all. I know it has been a long time and I would appreciate help. Most of the PRs are just simple config bumps or README updates. I honestly just got burned out responding to issues and maintaining an open source project. The current and historical issues reported generally center around common networking needs or misunderstanding on the design philosophy that drives Starscream.

The design goal behind Starscream has always been to make it extendable, but to keep the core API set to a minimal implementation of the Websocket RFC. An example of this philosophy was the removal of SSL pinning in the 4.0.0 release. SSL pinning is still easy to use in Starscream, but the former inclusion of SSL pinning took too much opinion on how to implement SSL pinning in an app and isn't apart of the core Websocket RFC. This change should have been better communicated and has led to some confusion, but again, the effort required to provide documentation and examples is a sizable time commitment.  

The biggest part of this design philosophy is responding to issues and PRs. The vast majority of the current and historical issues have been around certain app responsibilities like thread safety being reported as Starscream issues. It is a sizable time commitment to respond and support these issues and ensure app specific implementation details don't leak into the library implementation. 

TLDR; I would super appreciate the help and if you are excited to help support and update Starscream, please let me know!

**Comment by nsnull0** [comment]
Hi guys, let me know if you have created new repository extend from Starscream, I would love to have a look, but I also hope Starscream will still be usable and maintenance for the next 4 years (🤞 )

**Comment by Winglonelion** [comment]
@daltoniam But I think supporting the Silicon build system is necessary now bro. Any chance to release this early?

**Comment by acmacalister** [comment]
@daltoniam and I are feeling a bit more energized after taking a nice break. Going to work on a couple new features we have been kicking around (CI/CD for easy releases, better testing, TLS/Cert pinning docs and a thread safe implementation, etc). Going to close this out for now.
